pkg/server/commands/reverseexpand/reverse_expand_test.go

```diff
@@ -141,7 +141,116 @@ type document
 		} else {
 			require.Nil(t, res)
 		}
+		<-done
 	case <-done:
 		// OK!
 	}
 }
+
+// errorIterator is a mock iterator that returns error when calling next on the second Next call
+type errorIterator[T any] struct {
```

Comment:
Can we please move this to `internal/mocks` instead of inline here?

Reply:
done